README.md

```diff
@@ -1,2 +1,179 @@
-# Leaderboard
-This is a Leaderboard project
+<a name="readme-top"></a>
+
+<div align="center">
+
+<!-- MAIN HEADING -->
+
+  <h3><b>LeaderBoard Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Portfolio <a name="about-project"></a>
+
+> This is a Leaderboard project with just HTML, Javascript and CSS using ES6
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML
+2. Linters
+3. JavaScript ES6
+4. Webpack
+### Tech Stack <a name="tech-stack"></a>
+
+> This is a version with mono view Desktop 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">Linters</a></li>
+    <li><a href="https://reactjs.org/">Javascript</a></li>
```

Comment:
- [ ] To make you README looks more professional, I think it would be bettter if you add an appropriate link to each stack as all of them lead to the React.js documentation.

Reply:
Thanks a lot, the changes were implemented.